src/components/TableToolbar/HiddenFields.tsx

```diff
@@ -67,49 +88,113 @@ export default function HiddenFields() {
       colord(theme.palette.background.paper)
         .mix("#fff", theme.palette.mode === "dark" ? 0.16 : 0)
         .alpha(1);
-
     return (
-      <li {...props}>
-        <ColumnItem option={option}>
-          <Box
-            sx={[
-              { position: "relative", height: "1.5rem" },
-              selected
-                ? { color: "primary.main" }
-                : {
-                    opacity: 0,
-                    ".MuiAutocomplete-option.Mui-focused &": { opacity: 0.5 },
-                  },
-            ]}
-          >
-            <VisibilityIcon />
-            <IconSlash
-              sx={[
-                {
-                  "& .icon-slash-mask": {
-                    stroke: (theme) => slashColor(theme).toHslString(),
-                  },
-                  ".Mui-focused & .icon-slash-mask": {
-                    stroke: (theme) =>
-                      slashColor(theme)
-                        .mix(
-                          theme.palette.primary.main,
-                          theme.palette.action.selectedOpacity +
-                            theme.palette.action.hoverOpacity
-                        )
-                        .alpha(1)
-                        .toHslString(),
-                  },
-                },
-                selected ? { strokeDashoffset: 0 } : {},
-              ]}
-            />
-          </Box>
-        </ColumnItem>
-      </li>
+      <Draggable
+        draggableId={option.value}
+        index={option.index}
+        isDragDisabled={disableDrag}
+      >
+        {(provided) => (
+          <li {...props} ref={provided.innerRef} {...provided.draggableProps}>
+            <Box
+              sx={[{ position: "relative", height: "1.5rem" }]}
+              {...provided.dragHandleProps}
+            >
+              <DragIndicatorOutlinedIcon
+                color="disabled"
+                sx={[{ marginRight: "6px", opacity: disableDrag ? 0.6 : 1 }]}
```

Comment:
```suggestion
                sx={[
                  {
                    marginRight: "6px",
                    opacity: (theme) =>
                      disableDrag ? theme.palette.action.disabledOpacity : 1,
                  },
                ]}
```

There isn’t enough difference between the disabled and enabled icon’s opacities.